constants.py

```diff
@@ -193,6 +193,13 @@
 kLimelightTargetValidKey = "tv"
 kLimelightTargetHorizontalAngleKey = "tx"
 
+# Photonvision
+kPhotonvisionNetworkTableName = "photonvision"
+kPhotonvisionTargetValidKey = "hasTarget"
+kPhotonvisionTargetHorizontalAngleKey = "targetYaw"
+kPhotonvisionTargetVerticalAngleKey = "targetPitch"
+
+kPhotonvisionTargetSimDistanceKey = "simDistance"
```

Comment:
trusting you read the docs right lol

Reply:
At this point we know for sure that we're using two limelight's. This should no longer be photonvision